pkg/proxy/metrics/metrics.go

```diff
@@ -125,6 +125,18 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// SyncProxyRulesLastQueuedTimestamp is the last time a proxy sync was
+	// requested. If this is much larger than sync_proxy_rules_last_timestamp,
```

Comment:
Should measuring latency work better for detecting hungs?

Reply:
Latency can detect a node that tends to take a long time, which is valuable, but it's important to be able to alert on a node that is presently stale.

Reply:
In the comment it should be `sync_proxy_rules_last_timestamp_seconds`.

Reply:
yes, good point.